extensions/amp-carousel/0.2/test/test-e2e/test-arrows-in-lightbox.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {sleep} from './helpers';
+
+describes.endtoend(
+  'amp carousel in lightbox',
+  {
+    fixture: 'amp-carousel/0.2/amp-lightbox-carousel-selector.html',
+    environments: ['single'],
+  },
+  async (env) => {
+    let controller;
+    let nextArrow;
+    let prevArrow;
+
+    function getPrevArrow() {
+      return controller.findElement(
+        '.amp-carousel-button.amp-carousel-button-prev'
+      );
+    }
+
+    function getNextArrow() {
+      return controller.findElement(
+        '.amp-carousel-button.amp-carousel-button-next'
+      );
+    }
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    it('should open with both arrows', async () => {
+      // Click on image 2
+      const secondImage = await controller.findElement('#second');
+      await controller.click(secondImage);
+      // Wait for lightbox to load the carousel and image
+      await sleep(500);
```

Comment:
Is there some other way to know precisely when the lightbox has loaded?

Reply:
Yes, found it: `await expect(await controller.getElementProperty(lightbox, 'style')).to.not.be.null;`

Note that 
1.) Using hidden is not good here (checking for hidden to be an element property), since it's only added *after* it's closed,  so not good to check initially that it's opened
2.) You must you `await` for `controller.getElementProperty`, as with all controller methods, it returns a promise
- I found a few e2e tests that don't wait for this which probably means they're not doing what they should be.